python/ray/util/client/dataclient.py

```diff
@@ -81,6 +83,25 @@ def _start_datathread(self) -> threading.Thread:
             daemon=True,
         )
 
+    # A helper that takes in (request, callback) from queue, sets up tracking
+    # of the requests and yields the requests as a stream.
+    def _requests(self):
+        while True:
+            request = self.request_queue.get()
```

Comment:
you should try catch here. maybe sleep for 1 and blocking.

Reply:
What kind of exceptions need to be catched here?

Reply:
https://docs.python.org/3/library/queue.html#queue.Queue.get

I saw you check request None here. My point is that `get` will not return None.

Reply:
`None` is a special sentinel to signal shutdown of the data client, sent during `close()`.

Reply:
Got it. Thanks for the explanation!